trulens_eval/examples/frameworks/llama_index/llama_index_quickstart.ipynb

```diff
@@ -29,7 +29,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "#! pip install trulens-eval==0.12.0 llama_index==0.8.5.post2 html2text==2020.1.16"
+    "#! pip install trulens-eval==0.12.0 llama_index==0.8.29post1 html2text==2020.1.16"
```

Comment:
Currently this is commented out, but should we uncomment it?

Reply:
In general our template is to comment out this line to suggest that users primarily our maintaining a python environment and can use this line as a reference. Open to counter-arguments, really just a user flow thing.